session5/src/com/itschool/course/Course5PersonConstructor.java

```diff
@@ -0,0 +1,18 @@
+package com.itschool.course;
+
+public class Course5PersonConstructor {
+    public static void main(String) args) {
+       Person Andreea = new Person( "Marginas",  12434546567865754435L);
```

Comment:
I know that the names should be written with uppercase letters, but this is Java and not real world :)
so, if you don't mind, would you be kind to use Java notation for variables here and change this variable to start with lowercase "a"? :D